src/extension.ts

```diff
@@ -242,6 +265,7 @@ class RunLineInEditor {
 
     dispose() {
         this.disposables.forEach(disposable => disposable.dispose());
+        this.runStatusBarItem.dispose();
```

Comment:
Instead add the status bar item to `this.disposables`.

Reply:
Fixed